metc-wastewater-covid-monitor/server.R

```diff
@@ -1,546 +1,548 @@
+      mode = "lines",
+      x = ~date,
+      fill = "tozeroy",
+      y = ~copies_day_person_7day,
+      alpha = 0.25,
+      line = list(width = 0.5, color = colors$suppGray),
+      color = ~"Total load",
+      colors = pal,
+      name = "Total load",
+      hoverinfo = "text",
+      text = ~hover_text_load_7day
+    ) %>%
+    add_trace(
+      data = copies_by_variant,
+      type = "scatter",
+      mode = "lines",
+      x = ~date,
+      fill = "tozeroy",
+      y = ~copies_7day,
+      split = ~variant,
+      color = ~variant,
+      alpha = 0.25,
+      colors = pal,
+      hoverinfo = "none"
+    ) %>%
+    add_trace(
+      data = copies_by_variant,
+      type = "scatter",
+      mode = "markers",
+      x = ~date,
+      y = ~copies,
+      split = ~variant,
+      color = ~variant,
+      alpha = 0.8,
+      colors = pal,
+      hoverinfo = "text",
+      text = ~hover_text_variant,
+      showlegend = F
+    ) %>%
+    layout(
+      annotations = ann_list,
+      hovermode = "closest",
+      hoverdistance = "10",
+      hoverlabel = hov_lab_list,
+      margin = list(
+        l = 50,
+        r = 100,
+        b = 115,
+        pad = 10
+      ),
+      xaxis = list(
+        title = list(
+          text = "",
+          standoff = 25,
+          font = list(
+            size = 14,
+            family = font_family_list,
+            color = councilR::colors$suppBlack
+          )
+        ),
+        zerolinewidth = 2,
+        zeroline = TRUE,
+        showline = FALSE,
+        showgrid = FALSE,
+        tickfont = list(
+          size = 12,
+          family = font_family_list,
+          color = councilR::colors$suppBlack
+        )
+      ),
+      yaxis = list(
+        title = list(
+          text = "<b>Viral load in wastewater,</b> M copies/person/day",
+          standoff = 25,
+          font = list(
+            size = 14,
+            family = font_family_list,
+            color = councilR::colors$suppBlack
+          )
+        ),
+        tickfont = list(
+          size = 12,
+          family = font_family_list,
+          color = councilR::colors$suppBlack
+        ),
+        rangemode = "nonnegative",
+        gridcolor = "gray90",
+        zerolinecolor = "gray50",
+        zerolinewidth = 2
+      ),
+      legend = list(
+        orientation = "h",
+        y = -0.2,
+        xanchor = "left",
+        font = list(
+          size = 14,
+          family = font_family_list,
+          color = councilR::colors$suppBlack
+        )
+      )
+    ) %>%
+    config(displayModeBar = FALSE)
+
+  ## variant frequency -----
+  variantFreqPlot <-
+    # browser()
+    variant_data %>%
+    plot_ly() %>%
+    add_trace(
+      type = "scatter",
+      mode = "markers",
+      x = ~date,
+      y = ~frequency,
+      split = ~variant,
+      color = ~variant,
+      alpha = 0.8,
+      colors = pal,
+      hoverinfo = "text",
+      text = ~hover_text_variant,
+      showlegend = F

```

Comment:
This only hides the legend for the markers/points. Is this what we want? How does it look if we hide the legend for the lines?

Reply:
I can hide it for the lines instead of the markers and do another pull request.  Does that sound good?

Reply:
Do you want me to change it to markers for both variant plots?

Reply:
![image](https://user-images.githubusercontent.com/16826194/156825384-34178932-33a6-4a97-893d-2bac274848ce.png)

Nah, I think this looks good!